pkg/autoscaling/estimator/percentile.go

```diff
@@ -166,9 +184,22 @@ func getMemConfig(props map[string]string) *predictionconfig.Config {
 		marginFraction = "0.15"
 	}
 
+	initModeStr, exists := props["mem-model-init-mode"]
+	initMode := predictionconfig.ModelInitModeLazyTraining
+	if !exists {
+		initMode = predictionconfig.ModelInitMode(initModeStr)
+	}
+
+	historyLength, exists := props["mem-model-history-length"]
+	if !exists {
```

Comment:
Why history length of cpu and memory is not same, cpu is 24h but memory is 48h?

Reply:
> Why history length of cpu and memory is not same, cpu is 24h but memory is 48h?

This is just an empirical value borrowed from vpa.  memory is incompressible resource, so use longer history data is more safe and robust. cpu is compressible resource, and generally it is daily cycle because of people traffic

Reply:
got it.